src/js/redux/actions/lastDateModifiedActions.js

```diff
@@ -0,0 +1,4 @@
+export const setLastDateModifiedList = (state) => ({
+  type: 'SET_LASTDATEMODIFIED_LIST',
```

Comment:
I would recommend using underscores between each word to be consistent with other action names.